sources/ClangSharp.PInvokeGenerator/PInvokeGenerator.VisitDecl.cs

```diff
@@ -431,11 +431,13 @@ private void VisitFunctionDecl(FunctionDecl functionDecl)
                 type = attributedType.ModifiedType;
                 callConv = attributedType.Handle.FunctionTypeCallingConv;
             }
-            var functionType = (FunctionType)type;
 
-            if (callConv == CXCallingConv.CXCallingConv_Invalid)
+            if (type is FunctionType functionType)
             {
-                callConv = functionType.CallConv;
+                if (callConv == CXCallingConv.CXCallingConv_Invalid)
```

Comment:
nit: Simplify the nesting level here
```suggestion
if ((type is FunctionType functionType) && (callConv == CXCallingConv.CXCallingConv_Invalid))
```